hack/contrib/docker/gateway/nginxtmp/servers.tmpl

```diff
@@ -14,6 +14,7 @@ server {
 
     {{ if .SSLCertificate }}
     # ssl
+    ssl_protocols TLSv1.1 TLSv1.2 TLSv1.3;
```

Comment:
1. What is the OpenSSL version? Whether to support TLSv1.3?
2. Should a parameter be set to the configuration, which in some cases can be set by environment variables?

Reply:
ingress-controller default values is "TLSv1.2 TLSv1.3"

Reply:
fixed